docs/Github-workflow.md

```diff
@@ -0,0 +1,69 @@
+
+## Opening an issue
+When opening a new issue, use the appropriate template: Bug report, Feature request, or Report a security vulnerability. All new issues should include the following labels:
+
+- A [Focus] label, or Infrastructure if the issue relates to the plugin infrastructure. The [Focus] labels are aligned with the Performance team's [focus areas](https://docs.google.com/spreadsheets/d/16N5oZ9wE6AkiqMz7b_707eh24vvpjMwsEG67XFAbxy8/edit#gid=0) and [GitHub Projects](https://github.com/WordPress/performance/projects).
+- A [Type] label
+- A [Module] label if the issue relates to an existing module
+
+In addition, the new issue should be assigned to an appropriate Project: Object Cache, Measurement, Site Health, JavaScript, Infrastructure, or Images.
+
+By default, new issues will automatically be added to the Backlog column within its project. This is intended for tracking any future work that is not currently a high priority as defined by the project’s point(s) of contact (POCs). Contributors are welcome to work on issues in the Backlog, but response rates may be slower than they are on prioritized issues.
+
+### Working on an issue
+If you’re interested in working on an issue, please notify the POC for the issue’s focus area by tagging them on the issue and/or notifying them in the [weekly performance chat](https://make.wordpress.org/core/tag/performance/). For an updated list of POCs, please [see the `CODEOWNERS` file](https://github.com/WordPress/performance/blob/trunk/.github/CODEOWNERS).
+
+When you’re ready to begin work on an issue:
+- Assign it to yourself – Depending on your permissions levels, you may not be able to self-assign an issue. Please tag @bethanylang on an issue if you need assistance with permissions. Please only assign an issue if you plan to work on it within a reasonable time frame, i.e. within the next 2 weeks. 
+- Change the Project column to **In Progress**
+- Remove the **Needs Dev** label, if applicable
+
+## Needs Discussion
+Many issues require discussion and definition before development begins. For example, multiple approaches may be considered and the community should discuss them before a contributor proceeds with development. For those issues, add the **Needs Discussion** label and raise the issue in the weekly performance chat.
+
+## Needs Decision
+After discussion, a formal vote may be needed to determine how to proceed. If that’s the case, please tag @bethanylang in the issue for assistance with setting up a vote via GitHub comment. [An example vote can be found here](https://github.com/WordPress/performance/issues/92#issuecomment-1068215411).
+
+## Needs Dev
+If an issue requires (more) development and you are unable to complete it yourself, remove yourself as the assignee and add a **Needs Dev** label.
+
+## Needs Testing
```

Comment:
Shouldn't these be `h3` headings (`###`)? I think it doesn't make sense to have them at the same level as "Opening an issue". Maybe we can group them into another higher-level `h2` heading like "Workflow labels"?